main/src/library/Fixpoint/Ast/BodyPredicate.flix

```diff
@@ -30,15 +30,19 @@ namespace Fixpoint/Ast {
         pub def toString(body: BodyPredicate[v]): String =
             use Fixpoint.commaSeparate;
```

Comment:
Can we match on polarity and fixity separately and just build a string that way? So we avoid the exponential combination.